src/fragments/lib/logging/common/sending_logs/sending_logs.mdx

```diff
@@ -0,0 +1,19 @@
+With the Amplify Logger, you can send logs to Amazon CloudWatch from errors that are caught by the Amplify library or by adding your own custom log messages. For the Amplify library and custom logs you can customize which levels to log to CloudWatch.
```

Comment:
Small tweaks:

With the Amplify Logger, you can send logs to Amazon CloudWatch from errors that are caught by the Amplify library or by adding your own custom log messages. You can also customize which levels to log to send to CloudWatch.

Reply:
Updated text per suggestion.